src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2247,8 +2248,26 @@ private void ReportExceptionFallback(Exception e, string header)
             }
 
             // Add the position message for the error if it's available.
-            if (er != null && er.InvocationInfo != null)
+            if (er?.InvocationInfo is { })
+            {
                 Console.Error.WriteLine(er.InvocationInfo.PositionMessage);
+            }
+
+            // Print the stack trace.
+            var sb = new StringBuilder(capacity: 100)
+                .AppendLine()
+                .AppendLine($"--- {e.GetType().FullName} ---")
+                .AppendLine(e.StackTrace);
```

Comment:
Stack trace from related issue is ~2107 byte.
I believe we could set Capacity to 3 Kb here. (Or dynamically evaluate based on e.StackTrace.Length.)